classes/person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(id, age = 18, name = 'Unknown', parent_permission: true)
+    super()
     @id = id
```

Comment:
> - [OPTIONAL] Hello @Simpleshaikh1,  I suggest generating a random number as an ID instead of manually inputting it. I encountered some challenges while testing your Person class without passing an ID explicitly. This adjustment will benefit your implementation as well.👍
> 
> This is what I mean:
> 
> ```
> def initialize(age, name: 'Unknown', parent_permission: true)
>     super()
>     @id = Random.rand(1..1000) 
>     # Rest of your code
> end
> ```